webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -29,13 +41,37 @@ export default {
   },
   data() {
     return {
+      displayedNotifications: [],
       notifications: [],
     }
   },
   props: {
     placement: { type: String },
   },
+  watch: {
+    notifications: {
+      immediate: true,
+      handler(lastQueriedNotifications) {
+        if (lastQueriedNotifications && lastQueriedNotifications.length > 0) {
+          lastQueriedNotifications.forEach(updatedListNotification => {
+            const sameNotification = this.displayedNotifications.find(oldListNotification => {
+              return this.equalNotification(oldListNotification, updatedListNotification)
+            })
+            if (!sameNotification) this.displayedNotifications.unshift(updatedListNotification)
+          })
+        }
+      },
+    },
+  },
```

Comment:
@Tirokk you don't have to put this into a `watch` handler.

Vue-apollo has the `update` for graphql queries. We use it in a couple of other locations.  It's suitable for this task when you want to define **how** the `notifications` should be updated (in your case, you want to merge the new notifications with the old ones).

See here: https://vue-apollo.netlify.com/api/smart-query.html#options

We cannot use `fetchMore` as far as I know, because we're not using pagination here :thinking:

Reply:
Now it is so perfect that I can't even test my code anymore on my own.
Because if I write myself a notification it updates immediately at sending the comment. 
😂🤣😅😰 TOP !!!
But I could use two browsers ???

Reply:
Isn’t `refetch` enough?

Reply:
I was confused by `update` earlier: It triggers two times. Feel free to debug it with `console.log`. The first call comes when apollo reads from the cache. As soon as the new results from the server arrives, `update` is called again. It might be that apollo is so smart that it updates the cache if you mark a notification as `read`. :man_shrugging:

Reply:
Maybe `refetch` also works. I think `refetch` only triggers once. We use it for pagination.